code/modules/jobs/job_types/prisoner.dm

```diff
@@ -0,0 +1,81 @@
+/datum/job/prisoner
+	title = JOB_NAME_PRISONER
+	description = "As a prisoner your job is to be imprisoned. Play cards or chess, cook some food or grow some plants. Run away when security ain't looking."
+	department_for_prefs = DEPT_NAME_ASSISTANT
+	show_in_prefs = TRUE
+	faction = "Station"
+	exp_requirements = 300
+	exp_type = EXP_TYPE_CREW
+	total_positions = 3
+	min_pop = MINPOP_JOB_LIMIT
+	supervisors = "Security / The warden"
```

Comment:
```suggestion
	supervisors = "nobody"
```
Prisoners are not in the chain of command, security is not their supervisor and they don't have to listen to them